frontend/src/metabase/admin/settings/components/widgets/GroupMappingsWidget/GroupMappingsWidget.unit.spec.tsx

```diff
@@ -1,131 +0,0 @@
-import React from "react";
```

Comment:
If you create the container component, the testing of UI component will be easy and we won't need to remove specs